pkg/fftypes/tokenpool.go

```diff
@@ -28,6 +28,6 @@ type TokenPool struct {
 	Type       TokenType `json:"type"`
 	Namespace  string    `json:"namespace,omitempty"`
 	Name       string    `json:"name,omitempty"`
-	ProtocolID string    `json:"protocol_id,omitempty"`
+	ProtocolID string    `json:"protocolid,omitempty"`
```

Comment:
```suggestion
	ProtocolID string    `json:"protocolID,omitempty"`
```

Per discussion, the pattern in place is `ID` in capitals.